test/releases/0.13.0.go

```diff
@@ -0,0 +1,50 @@
+package releases
```

Comment:
I think you need to change this file I will like to see here and also in the 0.12.0 file the ovs and nmstate installation

Reply:
You are right, since we want to use this release in HCO with ovs and nmstate enabled by default, we should test it for future upgrades.

Reply:
Ok don't, can you guys verify ?